Intern page

vivianha534 opened the pull request:
**New files**
- Added intern page that has information about the Teach LA internship program

**Changed files**
- updated footer to include intern page
- updated join page to include a section about the internship program

vivianha534 commented:
#342

reginawang99 commented on intern.html:
nit: idk if we do this in the tla page, but when the string is this long, it's really hard for devs to read it in their VSCode or whatever!

(tldr add enters so that string isn't too hard to read, same with other lines in the PR)

feel free to dismiss!

reginawang99 replied:
also, the blobs of text on the page kinda make it hard to read :(( maybe we can add new lines between things at least?(and possibly links and stuffs?) idk tho your choice!

reginawang99 commented on intern.html:
```diff
@@ -0,0 +1,34 @@
+
+<h1 class="title page-title">Our Intern Program</h1>
+<h2 class="title">What is Teach LA's internship program?</h2>
+<div class="what-is-intern-container">
```
nit: maybe we can use something we already have? anything is good tho!
```suggestion
<div class="row-desktop row-desktop-ratio-1-2 row-desktop-reverse">
```

reginawang99 replied:
might need to swap the `p` tag and the `img` tag for this to work :)

reginawang99 commented on intern.html:
```suggestion
```

reginawang99 commented on _sass/_interns.scss:
```diff
@@ -0,0 +1,10 @@
+    }
+}
+
+.intern-img{
+    width: 40vw
+}
```
i dont think this is used anywhere?

reginawang99 commented on join.html:
```diff
@@ -171,6 +171,32 @@ <h2 class="title">Speaker Series</h2>
+                    <li><b>Leading curriculum meetings </b>to create curriculum for our partnered schools</li>
+                    <li><b>Leading event planning and ideation</b>to teach at schools</li>
+                </ol>
+            <p>Here's what we do:
+            </p>
+            <ul>
+                <li>Project lead "learning labs", interactive web apps that explain CS concepts to kids</li>
+                <li>Plan events like Day of Code, a one day event where we teach students about coding!</li>
+                <li>More in depth trainings and mentorship from Teach LA dev directors in a variety of technologies</li>
+            </ul>
```
IMO i think this is getting kinda long/messy, can we maybe just have the top one and have a link to past internship projects (which i think you are doing rn)? idk your choice tho!

reginawang99 commented on join.html:
```diff
@@ -171,6 +171,32 @@ <h2 class="title">Speaker Series</h2>
+        </div>
+    </div>
+    
+
```
```suggestion
```

reginawang99 reviewed (changes requested):
sick :)) just some nits!!

vivianha534 commented:
@reginawang99 added a couple more things (and took some stuff away) to the intern page to make it a little more readable.

**Changes**
- listed out difference between gen mems and interns
- added more images 
- removed some points about the intern program from join page

reginawang99 commented on intern.html:
```diff
@@ -0,0 +1,90 @@
+    </div>
+</div>
+<p>
+    To find out more about how you can join us as a general member check out our 
```
```suggestion
    To find out more about how you can join us as a general member, check out our 
```

reginawang99 reviewed (approved):
LGTM!! 🥳 

(i have some small typos/punctuations if you might want to fix tho!)

vivianha534 commented:
closes #342

vivianha534 commented:
closes #342
